<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>500</y>
      <width>131</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>otworz zdjecie</string>
    </property>
   </widget>
   <widget class="QLabel" name="imageLabel">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>50</y>
      <width>551</width>
      <height>411</height>
     </rect>
    </property>
    <property name="text">
     <string>zdjecie</string>
    </property>
   </widget>
   <widget class="QPushButton" name="maskButton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>500</y>
      <width>81</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>maska1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="maskButton_2">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>500</y>
      <width>81</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>maska2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="maskButton_3">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>500</y>
      <width>81</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>maska3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="original">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>500</y>
      <width>81</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>oryginal</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionactionExit">
   <property name="text">
    <string>actionExit</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionactionOtworz">
   <property name="text">
    <string>actionOtworz</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
